debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresSchemaIT.java

```diff
@@ -122,6 +124,22 @@ public void shouldLoadSchemaForExtensionPostgresTypes() throws Exception {
         }
     }
 
+    @Test
+    public void shouldLoadSchemaForPostgisTypes() throws Exception {
+        TestHelper.executeDDL("postgis_create_tables.ddl");
+        schema = new PostgresSchema(new PostgresConnectorConfig(TestHelper.defaultConfig().build()));
+        try (PostgresConnection connection = TestHelper.create()) {
+            schema.refresh(connection, false);
+            final String[] testTables = new String[] {"public.postgis_table"};
+            assertTablesIncluded(testTables);
+            Arrays.stream(testTables).forEach(tableId -> assertKeySchema(tableId, "pk", Schema.INT32_SCHEMA));
+
+            assertTableSchema("public.postgis_table", "p, ml",
+                              Geometry.builder().optional().build(), Geography.builder().optional().build());
+                              //SchemaBuilder.array(Geometry.builder().build()).optional().build());
```

Comment:
Why commented out?

Reply:
The array test was originally part of the other geometry tests, but I split it out so it could fail by itself. Will remove this.